docs/evm/ethereum-vs-flow-evm.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Differences between EVM on Flow and Ethereum
+sidebar_label: EVM on Flow vs. Ethereum
+sidebar_position: 14
+---
+
+# Differences between EVM on Flow and Ethereum
+
+## Introduction
+
+The Flow network uses [Cadence](https://cadence-lang.org/) as its main execution environment. Cadence offers a safer, more efficient, and more developer-friendly experience for building smart contracts and decentralized applications. However, supporting EVM could make it easier for existing applications and tools already deployed for the EVM ecosystem to onboard to the Flow network ([more](https://developers.flow.com/evm/about) on this topic).
```

Comment:
Lets hyperlink EVM and remove the brackets for conciseness.

Reply:
:-? not sure if I follow isn't it the bracket and url the way we construct hyperlinks in markdown?